.github/workflows/ci.yml

```
@@ -68,7 +68,11 @@ jobs:
             modin/engines/base/io/__init__.py \
             modin/engines/base/io/file_dispatcher.py
       - run: python scripts/doc_checker.py modin/utils.py
-      - run: python scripts/doc_checker.py modin/experimental/xgboost
+      - run: |
+          python scripts/doc_checker.py modin/experimental/xgboost/__init__.py \
+            scripts/doc_checker.py modin/experimental/xgboost/utils.py \
+            scripts/doc_checker.py modin/experimental/xgboost/xgboost.py \
+            scripts/doc_checker.py modin/experimental/xgboost/xgboost_ray.py
```

```suggestion
            modin/experimental/xgboost/utils.py \
            modin/experimental/xgboost/xgboost.py \
            modin/experimental/xgboost/xgboost_ray.py
```
`doc_checker` accepts a list of files, there's no need to check itself three times here :)